services/py-api/tests/unit_tests/test_handlers/test_participant_handlers.py

```diff
@@ -11,28 +11,22 @@
 from src.server.handlers.participants_handlers import ParticipantHandlers
 from src.server.schemas.request_schemas.schemas import ParticipantRequestBody
 from src.server.schemas.response_schemas.schemas import ErrResponse, ParticipantRegisteredInTeamResponse
-from src.service.participants_service import ParticipantService
 
 
 @pytest.fixture
-def participant_service_mock() -> Mock:
-    return Mock(ParticipantService, register_admin_participant=AsyncMock())
-
-
-@pytest.fixture
-def participant_handlers(participant_service_mock: Mock) -> ParticipantHandlers:
-    return ParticipantHandlers(participant_service_mock)
+def participant_handlers(participant_registration_service_mock: Mock) -> ParticipantHandlers:
+    return ParticipantHandlers(participant_registration_service_mock)
```

Comment:
Add a comment for where we get this argument because it is not imported. How does conftest.py work throughout the whole testing directory. Maybe add some more docs about how fixtures work

Reply:
https://github.com/AUBGTheHUB/monolith/blob/main/services/py-api/tests/unit_tests/README.md

Reply:
I updated the README in 2973c46fb277bc549a219ceb1d03bcceddbbb2f1